packages/cli/src/util/output/create-output.ts

```diff
@@ -17,7 +17,7 @@ export interface LogOptions extends PrintOptions {
 }
 
 export class Output {
-  private debugEnabled: boolean;
+  debugEnabled: boolean;
```

Comment:
Can we use `isDebugEnabled()` function instead?

Reply:
On second thought, this is only for tests right?

You can update the MockClient to accept options param and pass that through to the Output here

https://github.com/vercel/vercel/blob/b8114b8b39dd6067e68dee49a556dcc3320a31d8/packages/cli/test/mocks/client.ts#L26

Reply:
There is a mock `Client` object which gets its `output` property reset for each test. However, there's no way to specify whether the test should have debug enabled or not, so mutating the property on the mock `client.output` was the way we said to do it.

An alternative might be to have a proper `MockOutput` subclass, but actually `Output` isn't even a real class at the moment, so I think doing this is fine in this case and we could refactor at some point in the future (I'd _like_ for `Output` to be a proper class for example).